fsm/fsm/fsm_test.go

```diff
@@ -241,24 +241,6 @@ func TestMustNewFSM_State_Entry_Conflict_Panic(t *testing.T) {
 	t.Errorf("did not panic on initialize with conflict in entry state")
 }
 
-func TestMustNewFSM_State_Final_Not_Found_Panic(t *testing.T) {
```

Comment:
Why this test was removed?

UPD. Seems like this test was removed cause the test fails without the `len(f.finStates) == 0` check above.